packages/fx-core/src/plugins/resource/simpleauth/utils/common.ts

```diff
@@ -107,13 +107,27 @@ export class Utils {
       Constants.AadAppPlugin.id,
       Constants.AadAppPlugin.configKeys.teamsWebAppId
     ) as string;
-    const endpoint = Utils.getConfigValueWithValidation(
-      ctx,
-      isLocalDebug ? Constants.LocalDebugPlugin.id : Constants.FrontendPlugin.id,
-      isLocalDebug
-        ? Constants.LocalDebugPlugin.configKeys.endpoint
-        : Constants.FrontendPlugin.configKeys.endpoint
-    ) as string;
+
+    let endpoint: string;
+    if (!isArmSupportEnabled()) {
```

Comment:
Seems some kind of duplicated here? Maybe better to check isLocalDebug first?

Reply:
I think using the isArmSupportEnabled first can make the logic clear since we will remove the no arm support case in the future refactor.

Reply:
Agree. Good point.